src/thunderbots/firmware/main/main.c

```diff
@@ -338,10 +338,10 @@ static void run_normal(void) {
 		bool symbol_rate;
 		uint16_t pan;
 	} mrf_profiles[4] = {
-		{ 23, false, 0x1846U },
-		{ 23, false, 0x1847U },
-		{ 23, false, 0x1848U },
-		{ 23, false, 0x1849U },
+		{ 24, false, 0x1846U },
+		{ 24, false, 0x1847U },
+		{ 24, false, 0x1848U },
+		{ 24, false, 0x1849U },
```

Comment:
I gotta ask, why?

Reply:
We were assigned this channel at competition.

Reply:
Just as a thought, perhaps we could make this a constant?